robotiq_c_model_control/src/robotiq_c_model_control/robotiq_gripper_action_interface.py

```diff
@@ -258,12 +324,20 @@ def reset(self):
             goal = CModel_robot_output()
             goal.rACT = 0
             self.send_gripper_command(goal, parse=False)
+
+            # Wait for the gripper to deactivate
+            timer = RobotiqCommandTimeout(seconds=3.0)
+            timer.start()
             while not rospy.is_shutdown() and not self.interrupted:
                 with self.status_cv:
                     self.status_cv.wait(0.25)
                     status = deepcopy(self.last_status)
                 if status.gSTA == 0:
                     break
+                if timer.expired():
```

Comment:
Can you create a new Jira Task to explore if we can get away with retrying the reset command as opposed to shutting down the node and waiting for runtime manager to restart it. In the seems of things I may have been over aggressive